### DISCUSSION TITLE
Can i get a link

### DISCUSSION THREAD

**Discussion by stefs22** [streamlink/streamlink#5144] [Q&A] [answered]
### Description

I get it to work on vlc, but i want to get a link to put this to my cctv sistem, advice?

**Comment by bastimeyer** [comment, marked as answer]
- https://streamlink.github.io/cli.html#cmdoption-player-external-http
- https://streamlink.github.io/cli.html#cmdoption-stdout

**Comment by stefs22** [comment]
Thank you but sorry my english is not perfect and i dont understand what i need to do.
Can i get sample?
For example this: 

> https://www.youtube.com/watch?v=1-iS7LArMPA&ab_channel=EarthCam

**Reply by bastimeyer** [reply]
I have posted two of many different ways of how you can make other applications read streamlink's output. How you make those other applications read that output depends on your use case, which is out of scope of this discussion forum, and you'll have to figure this out on your own.
